corehq/apps/app_manager/views.py

```diff
@@ -1716,6 +1717,8 @@ def rename_action_case_type(mod):
             app.enable_module_filtering and
             should_edit('module_filter')):
         module['module_filter'] = request.POST.get('module_filter')
+        if '#user' in module['module_filter'] and not is_usercase_in_use(domain):
```

Comment:
I don't think you need the `is_usercase_in_use` check since that's already checked in `enable_usercase`